galaxy_ng/app/api/ui/serializers/user.py

```diff
@@ -31,13 +31,19 @@ class Meta:
             'groups',
             'password',
             'date_joined',
-            'is_superuser'
+            'is_superuser',
         )
         extra_kwargs = {
             'date_joined': {'read_only': True},
             'password': {'write_only': True, 'allow_blank': True, 'required': False}
         }
 
+    def auth_provider(self, user):
```

Comment:
This function is correct, but it's not hooked up to return anything when the API endpoint is called. Do some research on the `SerializerMethodField` field.